1-data-structures/3-python/4-quicksort.MD

```diff
@@ -0,0 +1,15 @@
+QuickSort is a popular sorting algorithm that follows the divide-and-conquer paradigm. It works by selecting a 'pivot' element from the array and partitioning the other elements into two sub-arrays according to whether they are less than or greater than the pivot. The sub-arrays are then recursively sorted.
```

Comment:
make this readme file with #Definition and add some other thing which are  mentioned in the issue
using proper way to write readme file